Makefile

```diff
@@ -52,7 +52,7 @@ darglint: deps
 
 .PHONY: mypy
 mypy: deps
-	poetry run mypy .
+	poetry run mypy --namespace-packages --explicit-package-bases .
```

Comment:
Those two are needed to make mypy consider the alembic folder as a module.
See https://mypy.readthedocs.io/en/stable/running_mypy.html#mapping-file-paths-to-modules